oecologia-australis.csl

```diff
@@ -608,9 +620,9 @@
         <text macro="locators"/>
         <group>
           <choose>
-            <if type="chapter" match="all" variable="edition">
-              <text variable="edition" prefix=" " suffix="ª "/>
-              <label variable="edition" form="short"/>
+            <if type="chapter book" match="all" variable="edition">
```

Comment:
this doesn't work. A test for `type="chapter book" match="all"` is _never_ true as an item can't be book and chapter at once But I don't think you need the test for edition here. just do 

```
<if type="chapter book" match="any">
  <group prefix=" " delimiter=" ">
    <text variable="edition"/>
    <label variable="edition" form="short"/>
  </group>
</if>
```

Reply:
The information about edition needs to be exhibited in both cases, when reference is chapter or a book. I will correct match by "any".